releasenotes/notes/sparselabelop-class-57ffd57121dadfe5.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds :class:`.SparseLabelOp` which maps sparse labels to coefficient values. This will form a base class for other operator types in future.
```

```suggestion
    Adds :class:`.SparseLabelOp` which maps sparse labels to coefficient values.
```
This change will actually be done before the next release, meaning that once these release notes get compiled together, this would be a bit confusing :+1: More details will be provided by the release notes which refactor the other operator classes